src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.cs

```diff
@@ -374,7 +390,7 @@ protected override void OnFontWeightChanged(FontWeight oldValue, FontWeight newV
 
 		partial void OnForegroundColorChangedPartial(Brush newValue);
 
-		#region PlaceholderText DependencyProperty
+#region PlaceholderText DependencyProperty
```

Comment:
While useful, avoid mixing cosmetic changes with your feature PR. You can make another PR for this, though.